cng/dsa.go

```diff
@@ -288,7 +284,7 @@ func encodeDSAKey(h bcrypt.ALG_HANDLE, params DSAParameters, X, Y BigInt) (bcryp
 		copy(blob, (*(*[sizeOfDSAV2BlobHeader]byte)(unsafe.Pointer(&hdr)))[:])
 		data := blob[sizeOfDSAV2BlobHeader:]
 		if err := encodeBigInt(data, []sizedBigInt{
-			{dsaSeedNil[:], groupSize},
+			{dsaSeedNil[:groupSize], groupSize},
```

Comment:
What is this for?

Reply:
This shouldn't have made into this PR, will revert it.